pages/api/send-reset-password-email.js

```diff
@@ -0,0 +1,72 @@
+import withContainer from "../../src/middleware/withContainer";
```

Comment:
Could you add a unit test spec for this API please?

Reply:
have added this 🙂